libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -335,6 +344,15 @@ bool AP_Arming::battery_checks(bool report)
         }
     }
 
+    if (((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_BATTERY)) && 
+        (_min_voltage > 0.0f)) {
+        if (_battery.voltage() < _min_voltage) {
```

Comment:
I'm thinking ARMING_VOLTAGE1 and 2 where both are 0 by default. Gives
flexibility.

On Fri, Feb 12, 2016 at 12:53 AM, WickedShell <notifications@github.com>
wrote:

> In libraries/AP_Arming/AP_Arming.cpp
> <https://github.com/diydrones/ardupilot/pull/3604#discussion_r52717195>:
>
> > @@ -335,6 +344,15 @@ bool AP_Arming::battery_checks(bool report)
> >          }
> >      }
> >
> > +    if (((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_BATTERY)) &&
> > +        (_min_voltage > 0.0f)) {
> > +        if (_battery.voltage() < _min_voltage) {
>
> Alright I can reimplement. I wasn't sure how the battery logic worked, and
> was wondering if there was a scenario where as long as the battery that was
> marked primary was fine that we would be happy to arm for flight. But I've
> never actually tried to use the multiple battery stuff, so I'm fine with
> changing it to match the behavior you're looking for.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3604/files#r52717195>.
>